src/Filter.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Filters } from './types/Filters';
+
+interface Props {
+  setFilter: (filter: Filters) => void;
+  filter: Filters;
+}
+
+export const Filter: React.FC<Props> = ({ filter, setFilter }) => {
+  const linkData = [
+    { label: 'All', value: Filters.ALL },
+    { label: 'Active', value: Filters.ACTIVE },
+    { label: 'Completed', value: Filters.COMPLETED },
+  ];
```

Comment:
1. Create static data outside react components.
2. Do you need this array? You can take the label from the enum's key or value.